File: Fun_03-06 to 03-20 mock/12646_Amanda.py
'''
OnlineJudge
12646 - Zero or One
Code by: Amanda Lim
'''

while True:
    try:
        input_ = input()
        a,b,c = input_.split(' ')
        if ((a == b) and (a == c)):
            print('*')
        if ((a != b) and (b == c)):
            print('A')
        if ((b != a) and (a == c)):
            print('B')
        if ((c != a) and (a == b)):
            print('C')
    except EOFError:
        break

File: Fun_03-06 to 03-20 mock/12646_Amanda.cpp
/*
*OnlineJudge
*12646 - Zero or One
*Code by: Amanda Lim
*/

#include <bits/stdc++.h>
using namespace std;

int main ()
{
    int A, B, C;
    while (cin >> A >> B >> C) {
        if ((A == B) && (B == C))
            cout << "*" << endl;
        if ((A != B) && (B == C))
            cout << "A" << endl;
        if ((A != B) && (A == C))
            cout << "B" << endl;
        if ((A != C) && (B == A))
            cout << "C" << endl;
    }
    return 0;
}